src/scripts/createTransaction.ts

```diff
@@ -7,37 +7,34 @@ import RahaStellar, { getNewTestAccount } from '../RahaStellar';
 const modes = ['test', 'prod'];
 
 function errorOnUsage() {
-    console.log('Usage is: node uploadBlock.ts [test|prod] [multiHash] [secretKey (optional if testing)]')
-    process.exit(1);
+    throw new Error('Usage is: node uploadBlock.ts [test|prod] [multiHash] [secretKey (optional if testing)]');
 }
 
 async function main(args) {
-    // Verify existence of isTest and multiHash arguments
-    if (args.length < 4) errorOnUsage();
-
-    const mode = process.argv[2];
-    if (!modes.includes(mode)) errorOnUsage();
-    const isTest = mode === 'test';
-
-    const multiHash = process.argv[3];
-
-    let publicKey;
-    let secretKey;
-    if (args.length < 5) {
-        if (isTest) {
-            const keyPair = await getNewTestAccount();
-            console.log(`New test public key is ${keyPair.publicKey()}.`);
-            secretKey = keyPair.secret();
+    try {
+        // Verify existence of isTest and multiHash arguments
+        if (args.length < 4) errorOnUsage();
+
+        const mode = process.argv[2];
+        if (!modes.includes(mode)) errorOnUsage();
+        const isTest = mode === 'test';
+
+        const multiHash = process.argv[3];
+
+        let publicKey;
+        let secretKey;
+        if (args.length < 5) {
+            if (isTest) {
+                const keyPair = await getNewTestAccount();
+                console.log(`New test public key is ${keyPair.publicKey()}.`);
```

Comment:
Can we print the test secret key? We don't care about losing test secret keys and could be useful to submit multiple blocks to same test account and then test out if laboratory is working well